int32 cmd_test
int32 vel_test
int32 time_test
